pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -63,11 +63,11 @@ func (l *ScrapeConfigList) DeepCopyObject() runtime.Object {
 
 // ScrapeConfigSpec is a specification of the desired configuration for a scrape configuration.
 type ScrapeConfigSpec struct {
-	StaticConfigs []StaticConfig     `json:"staticConfigs,omitempty"`
-	FileSDConfigs []FileSDConfig     `json:"fileSDConfigs,omitempty"`
-	HTTPSDConfigs []HTTPSDConfig     `json:"httpSDConfigs,omitempty"`
-	Relabelings   []v1.RelabelConfig `json:"relabelings,omitempty"`
```

Comment:
I remember you had an opinion on this naming: https://github.com/prometheus-operator/prometheus-operator/pull/5279#discussion_r1095902263

From what I see in podmonitor/probe, it seems that the name is `RelabelConfigs` but the json field name is `relabelings`:
* https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/podmonitor_types.go#L137
* https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/probe_types.go#L142

With that in mind, do you agree with the naming below or should it be named `Relabelings`?

Reply:
I'm less concerned by the Go fields than by the YAML field names.